players.js

```diff
@@ -236,37 +236,63 @@ class Player {
 		return ""
 	}
 	getArtUrlIcon(size){
+		const settings = ExtensionUtils.getSettings('org.gnome.shell.extensions.mpris-label');
```

Comment:
All of this should be done on `extension.js` then passed to the function as an argument, the same way as `size`

Reply:
Do you mean the padding-left and padding-right values passed as argument?

Reply:
The setting values should be passed from `extension.js` instead of having to call `ExtensionUtils.getSettings` here.

`getIcon` does the same though, that might be an oversight or means that isn't possible.

Reply:
Yeah that's why I did it but I moved them away now, so it's all in extension.js